text: mention that index is required for signing (#64)
BlockstreamResearch/bip-frost-dkg: README.md

In that case, the DKG output is a triple consisting of a *secret share* for participating in FROST signing sessions (individual to each participant, not returned to the coordinator), the *threshold public key* representing the `t`-of-`n` policy of the group (common to all participants and the coordinator), and a list of `n` *public shares* for verification of individual contributions to a FROST signing session (common to all participants and the coordinator).
Moreover, all parties obtain *recovery data* (common to all participants and the coordinator), whose purpose is detailed in the next subsection.

To participate in the FROST signing protocol, signers need their DKG output and their index in the host public key list, although the full list of host public keys is not required for signing.

> Additionally, to initiate a signing session, the signer or coordinator must know the indices of all participating signers within the host public key list.

Reviewer: When I read this sentence, I asked myself "which signer (or coordinator)"? I think who exactly will need it depends a bit on i) which FROST variant is used (Lategrange or not), and ii) how the application organizes things. I assume this is what you want to convey with "signer *or* coordinator"?

Perhaps that's a bit simpler then:
```suggestion
Additionally, the set of indices of all participating signers within the host public key list is required to initiate a signing session.
```

If you stick to the current text, I think s/signer/participant is a bit more consistent. While a participant will also be a signer, we currently stick to "participant" throughout the text.

Author: Agreed. Replaced the sentence with your suggestion.
